lib/engine/corporation.rb

```diff
@@ -157,11 +157,7 @@ def holding_ok?(share_holder, extra_percent = 0)
     end
 
     def all_abilities
-      all = @companies.flat_map(&:all_abilities)
-      @abilities.each do |ability|
-        abilities(ability.type) { |a| all << a }
-      end
-      all
+      @companies.map(&:all_abilities).flatten
```

Comment:
why was this changed? you've removed corporation abilities here?

Reply:
That's a mistake. Will fix. Surprised it passed all the tests!